doc/src/configuration.md

```diff
@@ -1,10 +1,10 @@
 # Configuration
 
-Rustup has a settings file in [TOML](https://github.com/toml-lang/toml) format
-at `${RUSTUP_HOME}/settings.toml`. The schema for this file is not part of the
-public interface for rustup - the rustup CLI should be used to query and set
-settings.
+Rustup has a [TOML](https://github.com/toml-lang/toml) settings file at
+`${RUSTUP_HOME}/settings.toml`. The schema for this file is not part of the
+public interface for rustup - the rustup CLI should be preferred to query and
+set settings.
 
-On Unix operating systems a fallback settings file is consulted for some
-settings. This fallback file is located at `/etc/rustup/settings.toml` and
-currently can define only `default_toolchain`.
+On Unix systems, by default, it is located at `${HOME}/.rustup/settings.toml`,
+and a fallback settings file at `/etc/rustup/settings.toml` is available, but
+can currently only define `default_toolchain`.
```

Comment:
This change makes the meaning less clear I feel. There are two meanings included in this paragraph now:

- That there is a fallback settings file at a certain location.
- That the location for the non-public settings file is is at a particular location.

I am not keen on documenting the location of the non-public settings file with any more clarity. We don't want people accessing it except when debugging a problem the CLI cannot cope with. If we really want to add this exposition, please keep the explanation in the paragraph about the non-public settings file, rather than in the paragraph about the public machine-wide settings file.

Reply:
I'm not aware of that "non-public" means in your message, I don't know much about this file because this `configuration` section isn't very informative at the moment.

I do think that omitting the location is bad for users, and would prefer to be told the location, plus a  and "don't change it"

Reply:
That being said, you recommend I revert all the changes in the second paragraph and add the location piece of information to the first one?

Reply:
yes